tests/login.test.js

```diff
@@ -0,0 +1,52 @@
+//import $ from 'webdriverio/build/commands/browser/$';
+import loginPage from '../pages/login.page';
+
+describe('OpenBank', () => {
+
+    /* it('Should show login page', async () => {
+
+      browser.url('login');
+	  
+	  await browser.pause(5000);
+    }); */
+    it('Should write in username and password and enter the home page', async () => {
```

Comment:
This name is written too procedurally (focusing on actions you are performing rather than on **expected results** after some **operation / data input**).

Prefer something like: "**Should** redirect to home page **when** attempting login with valid credentials".

Notice a couple of things regarding this recommendation:

- There is some implied structure imposed by the usage of **should** (for describing results) and then **when** (for describing actions).
- The language is more **domain oriented** (it abstracts out the details of what it really means to login), and emphasizes the state the data have to be in (credentials need to be valid)